packages/lib/tests/document.test.ts

```diff
@@ -5,100 +5,77 @@ import * as t from '../src/types'
 
 describe('document', () => {
```

Comment:
Is there any blessed library out there for validating a did doc?  I poked around but didn't turn one up.  Just could see it being useful for these tests, to ensure some level of compatibility.

Reply:
actually this seems like the closest thing: https://www.npmjs.com/package/@did-core/did-ld-json

i'll look at hooking it up 🧐

Reply:
eh this is not helpful at all & I'm like 98% sure I have the formatting correctly. 

fortunately, changing our did doc representation is an easy change to maek

Reply:
Sweet, sounds good!